techniques/css/C42.html

```diff
@@ -18,67 +18,87 @@ <h2>When to Use</h2>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this technique is to ensure that links in navigation or pagination menus will be spaced so that they fall within an area that measures at least 44 x 44 CSS pixels if the target area itself is smaller than that. The aim is to provide an adequate target clearance so the offset to adjacent targets is sufficient to prevent accidental pointer activation of adjacent targets.</p>	
-		
+			<p>The objective of this technique is to ensure that navigation or pagination links will be spaced so that they fall within an area that measures at least 44 &#215; 44 CSS pixels if the target area itself is smaller than that. The aim is to provide an adequate target clearance so the offset to adjacent targets is sufficient to prevent accidental pointer activation of adjacent targets.</p>
 		</section>
+		
 		<section id="examples">
 			<h2>Examples</h2>
-			<p>The following examples can be seen as rendered versions in the <a href="../../working-examples/css-pointer-spacing/">working examples</a>.</p>
+			<p>The following examples can be seen as <a href="../../working-examples/css-sufficient-target-spacing/">working examples</a>.</p>
```

Comment:
"following examples" > "following example" (singular, as we're going to remove the first one)